crates/driver/src/domain/competition/auction.rs

```diff
@@ -112,6 +113,32 @@ impl Auction {
     pub fn score_cap(&self) -> Score {
         self.score_cap
     }
+
+    /// All auction prices normalized to native token price.
+    ///
+    /// WEI to ETHER conversion
+    pub fn normalized_prices(&self) -> NormalizedPrices {
+        let mut prices = self
+            .tokens
+            .0
+            .iter()
+            .filter_map(|(address, token)| {
+                token.price.map(|price| {
+                    (
+                        *address,
+                        auction::NormalizedPrice(price.0 .0.to_big_rational() / &*UNIT),
+                    )
+                })
+            })
```

Comment:
nit: with `.chain()` you can immediately collect into the finished hashmap.
```suggestion
            })
            .chain([(
                eth::ETH_TOKEN,
                auction::NormalizedPrice(BigRational::from_integer(1.into())),
            )])
```

Reply:
added